src/index.js

```diff
@@ -20,6 +21,7 @@ root.render(
         <Route path='/home' element={<HomePage />} />
         <Route path='/products/:productName' element={<Product />} />
         <Route path='/shop' element={<ShopPage />} />
+        <Route path='/contact' element={<Contact />} />
```

Comment:
so you have:
- `HomePage`
- `Product`
- `ShopPage`
- `Contact`
why not call things consistently? it looks like 2 different persons wrote this code 6 months apart. if they're all pages call them the same everywhere: `ProductPage`, `ContactPage`